nodejs/week2/app.js

```diff
@@ -0,0 +1,71 @@
+import express from "express";
+import items from "./documents.json" assert { type: "json" };
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  res.send("This is a search engine");
+});
+
+app.get("/search", (req, res) => {
+  const query = req.query.q;
+
+  if (!query) {
+    res.status(200).json(items);
+  } else {
+    const match = items.filter((item) =>
+      Object.values(item).some(
```

Comment:
you repeat this many times in the code, so maybe it would be nice to extract this to a separate utility function? :)